web/src/components/Header/HeaderMenu.tsx

```diff
@@ -0,0 +1,70 @@
+import {QuestionCircleOutlined} from '@ant-design/icons';
+import {Popover} from 'antd';
+import {Dispatch, SetStateAction, useMemo} from 'react';
+import {useParams} from 'react-router-dom';
+import {DOCUMENTATION_URL, GITHUB_URL} from '../../constants/Common.constants';
+import * as S from './Header.styled';
+import {ShowOnboardingContent} from './ShowOnboardingContent';
+
+interface IProps {
+  visible: boolean;
+  pathname: string;
+  setIsOpen: Dispatch<SetStateAction<Boolean>>;
+  setVisible: Dispatch<SetStateAction<boolean>>;
+  onGuidedTourClick: () => void;
+  title: () => JSX.Element;
+}
+
+export const HeaderMenu = ({pathname, setIsOpen, title, visible, setVisible, onGuidedTourClick}: IProps) => {
+  const handleGuidedTourCLick = () => setVisible(o => !o);
+  const content = useMemo(
+    () => ShowOnboardingContent(onGuidedTourClick, setIsOpen, setVisible),
+    [setIsOpen, setVisible, onGuidedTourClick]
+  );
+  const params = useParams();
+  return (
+    <Popover visible={visible} content={content} title={title} arrowContent={null}>
```

Comment:
Are we sure we want to wrap the whole NavMenu in this Popover?

Reply:
that where the popover is wrapper in figma